db/migrate/20201219223722_create_eras.rb

```diff
@@ -5,6 +5,7 @@ def change
       t.string :name
       t.date :start_date
       t.date :end_date
+      t.integer :week_number
```

Comment:
Eras should have a start_week and end_week

Reply:
Got iiiit!  Okay cool! I'll have that working today.

Reply:
Should be good to go!